### PR TITLE
Assign expense Travelperk

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: fyle/connector.rb [hunk 1]
```diff
@@ -1819,7 +1819,7 @@
       input_fields: lambda do
         [
           {
-            name: "email",
+            name: "q",
```
Comment (NileshPant1999) [inline fyle/connector.rb:1822]: why are we changing this ??

File: fyle/connector.rb [hunk 2]
```diff
@@ -2213,6 +2213,55 @@
...
+                "optional": false,
+              },
+            ],
+          },
+        ]
+      end,
+      execute: lambda do |connection, input_fields|
+        user = call(:get_user_profile, connection)
+        admin = call(:get_employee, connection, input_fields["data"][0]["assignee_user_email"])
+        if user['email'] != input_fields["data"][0]["assignee_user_email"] && admin
```
Comment (NileshPant1999) [inline fyle/connector.rb:2246]: `if user['email'] != input_fields["data"][0]["assignee_user_email"]`
this condition will be sufficient no ??

File: fyle/connector.rb (hunk 1 above)
Comment (ruuushhh) [reply fyle/connector.rb:1822]: the new get employee api has `q` as a filter 
will add screenshot here
image

File: fyle/connector.rb (hunk 2 above)
Comment (ruuushhh) [reply fyle/connector.rb:2246]: No @NileshPant1999 as we will also have to check if the user exist in the same org or not

File: fyle/connector.rb (hunk 1 above)
Comment (NileshPant1999) [reply fyle/connector.rb:1822]: are that is okay but we can change it to something more reader usable, like query etc, as this will be displayed in UI

**Review by Shwetabhk — APPROVED** [review]

File: fyle/connector.rb (hunk 1 above)
Comment (ruuushhh) [reply fyle/connector.rb:1822]: yes got it @NileshPant1999

**Review by NileshPant1999 — APPROVED** [review]

File: fyle/connector.rb (hunk 1 above)
Comment (NileshPant1999) [reply fyle/connector.rb:1822]: why this is still q ??

File: fyle/connector.rb [hunk 3]
```diff
@@ -2213,6 +2213,55 @@
...
+                "type": "string",
+                "optional": false,
+              },
+            ],
+          },
+        ]
+      end,
+      execute: lambda do |connection, input_fields|
+        admin = call(:get_user_profile, connection)
+        user = call(:get_employee, connection, input_fields["data"][0]["assignee_user_email"])
```
Comment (NileshPant1999) [inline fyle/connector.rb:2245]: @ruuushhh this will fail if the assignee_user_email is null, you need to add a check on that

**Pull request merged**